doc/whats_new.rst

```diff
@@ -59,6 +59,8 @@ Detailed list of changes
 
 - The datatype in the dataframe returned by :func:`mne_bids.stats.count_events` is now
   ``pandas.Int64Dtype`` instead of ``float64``, by `Eric Larson`_ (:gh:`1227`)
+- The :func:`mne_bids.copyfiles.copyfile_ctf` now accounts for files with ``.{integer}_meg4`` extension, instead of only .meg4, 
+  when renaming the files of a .ds folder, by `Mara Wolter`_ (:gh:`1230`)
```

Comment:
you will also need to add yourself here: https://github.com/mne-tools/mne-bids/blob/main/doc/authors.rst